packages/logging-system/src/SetLocalStorage.ts

```diff
@@ -1,36 +1,32 @@
 import { LogRequestList, LogResponse, LogType } from './types/LogType';
 
 export const SetLocalStorageClear = () => {
-  const list: any[] = [];
+  const list: LogType[] = [];
   localStorage.setItem('yls-web', JSON.stringify(list));
 };
 
 export const SetLocalStorage = async (
   logger: LogType,
   putLog: (data: LogRequestList) => Promise<LogResponse>
 ) => {
-  if (window.localStorage.getItem('yls-web') == undefined) {
-    const list: any[] = [];
-    list.push(logger);
-    localStorage.setItem('yls-web', JSON.stringify(list));
-  } else {
-    const remainList: any[] = JSON.parse(localStorage.getItem('yls-web') as string) || [];
-    if (remainList.length < 10) {
-      const updateList = [...remainList, logger];
-      localStorage.setItem('yls-web', JSON.stringify(updateList));
-    } else {
-      const req: LogRequestList = {
-        logRequestList: remainList,
-      };
+  const list: LogType[] = JSON.parse(localStorage.getItem('yls-web') as string) || [];
+  list.push(logger);
+  localStorage.setItem('yls-web', JSON.stringify(list));
```

Comment:
```suggestion
  localStorage.setItem('yls-web.logs', JSON.stringify(list));
```
`yls-web.logs` 처럼 데이터의 정확한 명칭을 지정해서 의도를 명확히 하고, 앞으로 다른 데이터가 추가되어도 네이밍 문제가 생기지 않도록 하면 좋겠습니다.